### ISSUE TITLE
RancherOS shouldn't require network to retrieve console yaml

### ISSUE THREAD

**Issue by deniseschannon** [rancher/os#929]
**RancherOS Version: (ros os version)**
v0.4.5-rc2

When enabling a console, we shouldn't require the internet to retrieve the console yaml. 

Steps to Reproduce:

Install to Disk on VirtualBox
Enable ubuntu-console, which will download the console image.
Reboot with no internet.

RancherOS won't start up cause it's looking for the ubuntu-console yaml.

Note: If there is no network, we should just default back to the busybox console or perhaps a recovery console

**Comment by deniseschannon** [comment]
Attempted to test with same steps. 

When I booted up with no wifi, I ended up in the default console (instead of ubuntu-console), but I have 2 consoles in system-docker. 

```
[rancher@rancher ~]$ sudo system-docker ps -a
CONTAINER ID        IMAGE                          COMMAND                  CREATED             STATUS                        PORTS               NAMES
22ae4955f146        rancher/os-console:77612b8     "/usr/sbin/entry.sh /"   17 minutes ago      Up 15 minutes                                     console_754c57e55ebc
efae6982fd5d        rancher/os-console:77612b8     "/usr/sbin/entry.sh /"   17 minutes ago      Up 15 minutes                                     console
56f3d0628987        rancher/os-console:77612b8     "/usr/sbin/entry.sh /"   19 minutes ago      Exited (0) 17 minutes ago                         once_switch-console_1
5c1a4dbcb39e        rancher/os-docker:77612b8      "/usr/sbin/entry.sh /"   2 hours ago         Up 15 minutes                                     docker
3937075799b1        rancher/os-preload:77612b8     "/usr/sbin/entry.sh /"   2 hours ago         Exited (0) 15 minutes ago                         preload-user-images
ee20b7b20df7        rancher/os-base:77612b8        "/usr/sbin/entry.sh n"   2 hours ago         Up 15 minutes                                     network
94accb9ac831        rancher/os-cloudinit:77612b8   "/usr/sbin/entry.sh /"   2 hours ago         Exited (0) 15 minutes ago                         cloud-init
28ca6e772923        rancher/os-ntp:77612b8         "/usr/sbin/entry.sh /"   2 hours ago         Up 15 minutes                                     ntp
b9ff0fe8560e        rancher/os-base:77612b8        "/usr/sbin/entry.sh n"   2 hours ago         Exited (143) 15 minutes ago                       network-pre
0773f4c909d7        rancher/os-cloudinit:77612b8   "/usr/sbin/entry.sh /"   2 hours ago         Exited (0) 15 minutes ago                         cloud-init-pre
0f2912cb9e97        rancher/os-udev:77612b8        "/usr/sbin/entry.sh /"   2 hours ago         Up 15 minutes                                     udev
f0b627a88ef1        rancher/os-state:77612b8       "/usr/sbin/entry.sh e"   2 hours ago         Created                                           all-volumes
e9630198d824        rancher/os-acpid:77612b8       "/usr/sbin/entry.sh /"   2 hours ago         Up 15 minutes                                     acpid
7234dd8e02e5        rancher/os-preload:77612b8     "/usr/sbin/entry.sh /"   2 hours ago         Exited (0) 15 minutes ago                         preload-system-images
a69fe29b0e7a        rancher/os-udev:77612b8        "/usr/sbin/entry.sh /"   2 hours ago         Exited (0) 15 minutes ago                         udev-cold
272e8d80b2d3        rancher/os-syslog:77612b8      "/usr/sbin/entry.sh /"   2 hours ago         Up 15 minutes                                     syslog
e6670dd1dc71        rancher/os-state:77612b8       "/usr/sbin/entry.sh e"   2 hours ago         Created                                           container-data-volumes
e7bd31800265        rancher/os-state:77612b8       "/usr/sbin/entry.sh e"   2 hours ago         Created                                           user-volumes
4435346bbb69        rancher/os-state:77612b8       "/usr/sbin/entry.sh e"   2 hours ago         Created                                           command-volumes
e982fdb332dd        rancher/os-state:77612b8       "/usr/sbin/entry.sh e"   2 hours ago         Created                                           system-volumes
```
`

**Comment by deniseschannon** [comment]
Verified with custom build. Needs to be closed when #994 is merged.
